its/src/test/java/com/sonar/it/jenkins/orchestrator/JenkinsOrchestrator.java

```diff
@@ -329,9 +330,18 @@ public JenkinsOrchestrator newFreestyleJobWithScannerForMsBuild(String jobName,
 
     if (solutionFile != null) {
       findElement(buttonByText("Add build step")).click();
-      findElement(By.linkText("Build a Visual Studio project or solution using MSBuild")).click();
-      select(findElement(By.name("msBuildBuilder.msBuildName")), "MSBuild");
-      setTextValue(findElement(By.name("msBuildBuilder.msBuildFile")), solutionFile);
+      if (isDotNetCore) {
+        if (SystemUtils.IS_OS_WINDOWS) {
+          findElement(By.linkText("Execute Windows batch command")).click();
+          setTextValue(findElement(By.name("command")), "dotnet build " + solutionFile);
+        } else {
+          findElement(By.linkText("Execute shell")).click();
```

Comment:
I can't find what how to `setTextValue` for shell
![image](https://user-images.githubusercontent.com/11340282/41099668-75ef1af4-6a5f-11e8-8ec6-34ab090bebda.png)